### PR TITLE
Add geoglam collection

### PR DESCRIPTION
**Pull request by alukach**
As part of #127, here is the collection added for the GEOGLAM dataset.

### CODE REVIEW

File: data/collections/geoglam.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "id": "geoglam",
+  "type": "Collection",
+  "links": [],
```
Comment (alukach) [inline data/collections/geoglam.json:4]: Should we add any links?

Comment (abarciauskas-bgse) [reply data/collections/geoglam.json:4]: Yes please could we add https://cropmonitor.org/

**Review by abarciauskas-bgse — APPROVED** [review]

File: data/collections/geoglam.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "title": "GEOGLAM Crop Monitor",
+  "extent": {
+    "spatial": {
+      "bbox": [[-180, -90, 180, 90]]
+    },
+    "temporal": {
+      "interval": [["2020-01-01T00:00:00Z", "2022-04-30T23:59:59Z"]]
+    }
+  },
+  "license": "public-domain",
```
Comment (slesaad) [inline data/collections/geoglam.json:14]: we're using 'MIT' now

Comment (alukach) [reply data/collections/geoglam.json:14]:
```suggestion
  "license": "MIT",
```

**Pull request merged**